src/rpcserver.cpp

```diff
@@ -865,12 +865,6 @@ UniValue CRPCTable::execute(const std::string& strMethod, const UniValue& params
     if (!pcmd)
         throw JSONRPCError(RPC_METHOD_NOT_FOUND, "Method not found");
 
-    // Observe safe mode
-    string strWarning = GetWarnings("rpc");
-    if (strWarning != "" && !GetBoolArg("-disablesafemode") &&
-        !pcmd->okSafeMode)
-        throw JSONRPCError(RPC_FORBIDDEN_BY_SAFE_MODE, string("Safe mode: ") + strWarning);
```

Comment:
Any reason to keep `RPC_FORBIDDEN_BY_SAFE_MODE` in rpcprotocol.h?

Reply:
Looks like it is only used by

throw JSONRPCError(RPC_FORBIDDEN_BY_SAFE_MODE, string("Safe mode: ") + strWarning);

which is going away. I think it is safe to delete.

Reply:
Only reason is that it never occurred to me :)